server/app.js

```diff
@@ -109,13 +108,13 @@ app.use(methodOverride())
 
 /* database setup */
 let mongodb
-const mongoURI = getMongoURI({ settings: config.database.mongo })
+const mongoURI = process.env.MONGODB_URI
 const mongodbPromise = co(function* () {
   return yield MongoClient.connect(mongoURI, config.database.mongo.server)
 }).then(function (res) {
   mongodb = res.db()
   app.locals.appDb = res.db()
-  app.locals.dataDb = res.db(config.database.mongo.dataDB)
+  app.locals.dataDb = res.db(process.env.DATA_DB_NAME)
```

Comment:
I'm actually wondering if we need this as a env var or if we can hard-code it to `dpdata`. What do you think?

Reply:
@mikestone14 [2693cc2](https://github.com/AMP-SCZ/dpdash/pull/358/commits/2693cc23cae87aa48c4c47d52963d94881487249)
this is the README Commit

Reply:
yeah we can take it out